git_questions.md

```diff
@@ -0,0 +1,14 @@
+1. What is git?
+2. What is commit?
+3. What is branch?
+4. What are basic operations of git?
```

Comment:
Các hoạt động của git (em tha, khảo nhiều nguồn)
- git config: set user name và email trong main configuration file
- git init: khởi tạo một git ở local drive
- git clone: Sao chép repository đưa vào directory mới, tạo nhiều "branch theo dõi từ xa" cho từng branch trong repository được clone đó, tạo và kiểm tra một nhánh ban đầu được phân nhánh từ branch đang hoạt động của repository được clone. 
- git status: hiển thị toàn bộ trạng thái repository
- git stage: giai đoạn coi lại file chuẩn bị lưu.  
- git add: đưa nội dung file vào staging area
- git commit: lưu trạng thái file ("git commit-a" cho phép bỏ qua staging area và đưa thẳng vào repository)
- git push/pull: push đẩy các commit lên remote server /remote repository, pull dùng để cập nhật các thay đổi từ remote repo. Git pull là kết hợp của "git fetch" và "git merge", nghĩa là, khi dùng "git pull" ta nhận được cập nhật từ "remote repo (git fetch)" và ngay lập tức áp dụng thay đổi mới nhất đó vào "local repo (git merge)". Lệnh này cũng gây xung đột khi cần thực hiện các lệnh riêng rẽ.  (nguồn freecodecamp)
- git log: đưa ra danh sách commit được ghi trước commit nằm sau ký hiệu ^ (ví dụ "git log foo bar ^baz": nghĩa là liệt kê tất cả các commit có thể từ "foo" hoặc "bar" nhưng không phải từ "baz")  
- git branch: